argilla/docs/scripts/gen_popular_issues.py

```diff
@@ -79,52 +79,53 @@ def fetch_data_from_github(repository, auth_token):
 with mkdocs_gen_files.open(DATA_PATH, "w") as f:
     df = fetch_data_from_github(REPOSITORY, GITHUB_ACCESS_TOKEN)
 
-    open_issues = df.loc[df["State"] == "open"]
-    engagement_df = (
-        open_issues[["URL", "Issue", "Reactions", "Comments"]]
-        .sort_values(by=["Reactions", "Comments"], ascending=False)
-        .head(10)
-        .reset_index()
-    )
-
-    community_issues = df[df["Author association"] != "MEMBER"]
-    community_issues_df = (
-        community_issues[["URL", "Issue", "Created at", "Author", "State"]]
-        .sort_values(by=["Created at"], ascending=False)
-        .head(10)
-        .reset_index()
-    )
-
-    planned_issues = df[
-        ((df["Milestone"].str.startswith("v2")) & (df["State"] == "open"))
-        | ((df["Milestone"].str.startswith("2")) & (df["State"] == "open"))
-    ]
-    planned_issues_df = (
-        planned_issues[["URL", "Issue", "Created at", "Milestone", "State"]]
-        .sort_values(by=["Milestone"], ascending=True)
-        .head(10)
-        .reset_index()
-    )
-
-    f.write('=== "Most engaging open issues"\n\n')
-    f.write("    | Rank | Issue | Reactions | Comments |\n")
-    f.write("    |------|-------|:---------:|:--------:|\n")
-    for ix, row in engagement_df.iterrows():
-        f.write(f"    | {ix+1} | [{row['Issue']}]({row['URL']}) | 👍 {row['Reactions']} | 💬 {row['Comments']} |\n")
-
-    f.write('\n=== "Latest issues open by the community"\n\n')
-    f.write("    | Rank | Issue | Author |\n")
-    f.write("    |------|-------|:------:|\n")
-    for ix, row in community_issues_df.iterrows():
-        state = "🟢" if row["State"] == "open" else "🟣"
-        f.write(f"    | {ix+1} | {state} [{row['Issue']}]({row['URL']}) | by **{row['Author']}** |\n")
-
-    f.write('\n=== "Planned issues for upcoming releases"\n\n')
-    f.write("    | Rank | Issue | Milestone |\n")
-    f.write("    |------|-------|:------:|\n")
-    for ix, row in planned_issues_df.iterrows():
-        state = "🟢" if row["State"] == "open" else "🟣"
-        f.write(f"    | {ix+1} | {state} [{row['Issue']}]({row['URL']}) |  **{row['Milestone']}** |\n")
-
-    today = datetime.today().date()
-    f.write(f"\nLast update: {today}\n")
+    if len(df):
```

Comment:
In `main`, this was fixed but another way not to remove completely the tables/tabs.
```python
df["Milestone"] = df["Milestone"].astype(str).fillna("")
planned_issues = df[
    ((df["Milestone"].str.startswith("v2")) & (df["State"] == "open"))
    | ((df["Milestone"].str.startswith("2")) & (df["State"] == "open"))
]
```

Reply:
@sdiazlor sorry, is this a code suggestion?

Reply:
@davidberenstein1957 to avoid merge conflicts instead of the if statement, as it solves the error raised when no token is provided but keeps the basic the tabs/table format.